### PR TITLE
[hi] Add content/hi/docs/tutorials/kubernetes-basics/expose/expose-interactive.html

### PR DESCRIPTION
**Pull request by rajatgupta24**
Hey folks, this is a PR for hi localization of the file: 
`content/hi/docs/tutorials/kubernetes-basics/expose/expose-interactive.html `

PR is linked under this issue #29353

/language hi

### CODE REVIEW

**Comment by anubha-v-ardhan** [comment]
/retitle [hi] Add content/hi/docs/tutorials/kubernetes-basics/expose/expose-interactive.html

File: content/hi/docs/tutorials/kubernetes-basics/expose/expose-interactive.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: इंटरएक्टिव ट्यूटोरियल - अपने ऐप को उजागर करना
```
Comment (verma-kunal) [inline content/hi/docs/tutorials/kubernetes-basics/expose/expose-interactive.html:2]:
```suggestion
title: इंटरएक्टिव ट्यूटोरियल - अपने ऐप को एक्सपोज़ करना
```

Comment (verma-kunal) [reply content/hi/docs/tutorials/kubernetes-basics/expose/expose-interactive.html:2]: Do check the sheet [here](https://docs.google.com/spreadsheets/d/1UmL4K0v4yaN34k4WzYDZBobLBp5OZQp4GCTi28AfwQM/edit#gid=0) for words like these!! 
Some of them may have already been discussed & we encourage to use the same every where

File: content/hi/docs/tutorials/kubernetes-basics/expose/expose-interactive.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+weight: 20
+---
+
+<!DOCTYPE html>
+
+<html lang="en">
```
Comment (verma-kunal) [inline content/hi/docs/tutorials/kubernetes-basics/expose/expose-interactive.html:8]:
```suggestion
<html lang="hi">
```

Comment (verma-kunal) [reply content/hi/docs/tutorials/kubernetes-basics/expose/expose-interactive.html:8]: as we are doing Hindi Localization!!
We always encourage to refer other localizations, as you would get a much better idea on what to change!!

File: content/hi/docs/tutorials/kubernetes-basics/expose/expose-interactive.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+            <div class="katacoda__alert">
+                टर्मिनल का उपयोग करने के लिए, कृपया डेस्कटॉप या टैबलेट संस्करण का उपयोग करें
+            </div>
+            <div class="katacoda__box" id="inline-terminal-1"  data-katacoda-id="kubernetes-bootcamp/8" data-katacoda-color="326de6" data-katacoda-secondary="273d6d" data-katacoda-hideintro="false" data-katacoda-prompt="Kubernetes Bootcamp Terminal" style="height: 600px;">
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-12">
+            	<a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/explore/explore-intro/" role="button">&lt; मॉड्यूल 3 पर लौटें<span class=""></span></a>
+            	<a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/" role="button">होम <span class=""></span></a>
```
Comment (verma-kunal) [inline content/hi/docs/tutorials/kubernetes-basics/expose/expose-interactive.html:30]:
```suggestion
            	<a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/" role="button">होम<span class=""></span></a>
```

**Comment by anubha-v-ardhan** [comment]
@rajatgupta24 Can you change the base branch to `dev-1.22-hi.1`. That should make this PR ready to merge.

File: content/hi/docs/tutorials/kubernetes-basics/expose/expose-interactive.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+                टर्मिनल का उपयोग करने के लिए, कृपया डेस्कटॉप या टैबलेट संस्करण का उपयोग करें
+            </div>
+            <div class="katacoda__box" id="inline-terminal-1"  data-katacoda-id="kubernetes-bootcamp/8" data-katacoda-color="326de6" data-katacoda-secondary="273d6d" data-katacoda-hideintro="false" data-katacoda-prompt="Kubernetes Bootcamp Terminal" style="height: 600px;">
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-12">
+            	<a class="btn btn-lg btn-success" href="/hi/docs/tutorials/kubernetes-basics/explore/explore-intro/" role="button">&lt; मॉड्यूल 3 पर लौटें<span class=""></span></a>
+            	<a class="btn btn-lg btn-success" href="/hi/docs/tutorials/kubernetes-basics/" role="button">होम<span class=""></span></a>
+                <a class="btn btn-lg btn-success" href="/hi/docs/tutorials/kubernetes-basics/scale/scale-intro/" role="button">मॉड्यूल 5 जारी रखें &gt;<span class=""></span></a>
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tutorials/kubernetes-basics/expose/expose-interactive.html:31]:
```suggestion
                <a class="btn btn-lg btn-success" href="/hi/docs/tutorials/kubernetes-basics/scale/scale-intro/" role="button">मॉड्यूल 5 पर जाएं &gt;<span class=""></span></a>
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
@rajatgupta24 Final Review. Please incorporate the changes.
Please change the base branch to `dev-1.22-hi.1` for this PR to be merged.

**Comment by anubha-v-ardhan** [comment]
Thank you @rajatgupta24

**Pull request merged**